pom.xml

```diff
@@ -51,6 +51,7 @@
         <!--dependency versions -->
         <junit.version>5.0.0</junit.version>
         <junit.platform.version>1.0.0</junit.platform.version>
+        <testcontainers.version>1.6.0</testcontainers.version>
```

Comment:
testcontainers.version must be maintained by celesta to keep backward compatibility

Reply:
If I remove testcontainers.version, then Maven doesn't know which version of org.testcontainers:mssqlserver to choose, since we don't have such a dependency in Celesta (and cannot have, since we modified the class), and the build fails.

And then it becomes complicated: should I put the version 1.4.3 to mssqlserver dependency manually? 

Didn't quite get your point about backwards compatibility: you mean backwards compatibility with what? Does it matter if we use different versions of testcontainers for testing Celesta and Bass?